xgboost_ray/matrix.py

```diff
@@ -17,13 +17,18 @@
 
 import ray
 
+from xgboost_ray.util import Unavailable
+
 try:
     from ray.util.data import MLDataset
 except ImportError:
+    MLDataset = Unavailable
 
-    class MLDataset:
-        pass
-
+try:
+    import modin  # noqa: F401
+    MODIN_INSTALLED = True
+except ImportError:
```

Comment:
Do you have to set `modin = None` here?

Reply:
No, this is just to check if we can import it (`F401` is the warning that a module is imported but unused). We cannot actually import the real required modin modules (DataFrame, Series) here as it would initialize ray then.